src/data/complex/basic.lean

```diff
@@ -133,7 +133,7 @@ by simpa using @conj_inj z 0
 
 lemma eq_conj_iff_real {z : ℂ} : conj z = z ↔ ∃ r : ℝ, z = r :=
 ⟨λ h, ⟨z.re, ext rfl $ eq_zero_of_neg_eq (congr_arg im h)⟩,
- λ ⟨h, e⟩, e.symm ▸ rfl⟩
+ λ ⟨h, e⟩, by rw [e, conj_of_real]⟩
```

Comment:
Why does the old proof fail here?

Reply:
I made this change and I have no idea...

Reply:
For what it's worth, the error message was:
```
/mathlib/src/data/complex/basic.lean:136:18: error: "eliminator" elaborator type mismatch, term
  rfl
has type
  ?m_2 = ?m_2
but is expected to have type
  conj ↑h = ↑h
Additional information:
/mathlib/src/data/complex/basic.lean:136:18: context: the inferred motive for the eliminator-like application is
  λ (_x : ℂ), conj _x = _x
```

Reply:
I'm not sure why it worked before. It would need to prove `(-0 : real) = 0` by `rfl`, which shouldn't be possible if the reals are irreducible.